.projectile

```diff
@@ -0,0 +1 @@
+-/vendor
```

Comment:
Are you saying that I should remove this file from the PR? I don't have a problem with that, but just to clarify I intentially added it in the first place.